ztp/gitops-subscriptions/argocd/SNOExpansion.md

```diff
@@ -0,0 +1,483 @@
+# SNO expansion #
+
+SNO expansion is a scenario where initially a SNO is deployed into production, and at a later date the cluster size is increased by the addition of a worker node.
+The resulting cluster will contain two nodes:
+- Original SNO serving as the control plane for itself and the additional node
+- One additional node without any control plane components (controlled by the SNO).
+
+This transition incurs zero downtime on the original SNO node.
+Addition of more than one worker node to the SNO cluster is possible, but has not been verified in this version.
```

Comment:
i don't know of any limitations of adding as many workers as you want @omertuc correct me if i'm wrong

Reply:
My thought here was that adding a worker to SNO would incur some costs in CPU use by the control plane components. We are going to measure this soon and put a "CPU price tag" on each additional worker.  Maybe the wording should reflect that?

Reply:
That can help, from the wording it sounds like we support only adding one worker.

Reply:
Fixed